src/index.js

```diff
@@ -1 +1,45 @@
 import './style.css';
+import NewPlayer from './newplaya';
+import moveToApi from './api';
+
+export default class Ui {
+  static render(players) {
+    const listItems = document.querySelector('.list');
+    players.forEach((player) => {
+      listItems.innerHTML += `
+      <li>
+        <p>${player.user}:</p>
+        <p> ${player.score}</p>
+      </li>
+      `;
+    });
+  }
+
+  static clearField() {
+    const listItems = document.querySelector('.list');
+    listItems.innerHTML = '';
+  }
+}
+
+const getData = async () => {
+  const pull = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Ml0wfxegyFf06umHiVlI/scores/');
+
+  const data = await pull.json();
+  const dataOut = await data.result;
+  Ui.render(dataOut);
+};
```

Comment:
- [ ] [OPTIONAL] If you want to make your code look more professional you might want to try and keep the API calls together in 1 file like you already did with the `moveToApi()` function.